experimental/QuadFormAndIsom/src/spaces_with_isometry.jl

```diff
@@ -730,6 +730,27 @@ function Base.hash(V::QuadSpaceWithIsom, u::UInt)
   return Base.hash(isometry(V), u)
 end
 
+###############################################################################
+#
+#  Spinor norm
+#
+###############################################################################
+
+@doc raw"""
+    spinor_norm(Vf::QuadSpaceWithIsom; s::Int = -1) -> QQFieldElem
+
+Given a rational quadratic space with isometry $(V, b, f)$, return the real
+spinor norm of the extension of $f$ to $V\otimes \mathbb{R}$.
+
+If `s == -1`, then the spinor norm is computed with respect to $-b$.
+"""
+function spinor_norm(Vf::QuadSpaceWithIsom; s::Int = -1)
+  @req dim(Vf) > 0 "V must have positive dimension"
+  D, U = Hecke._gram_schmidt(gram_matrix(Vf), QQ)
+  fD = U*isometry(Vf)*inv(U)
+  return Oscar.spin(s*D, fD)
```

Comment:
```suggestion
  return spin(s*D, fD)
```
what is `spin` computing?

Reply:
The (rational) spinor norm of `fD` with respect to the diagonal Gram matrix `s*D`. The implementation only works for orthogonal basis, this is why I have to do a Gram-Schmidt orthogonalisation first.